docs/README.md

```diff
@@ -119,13 +119,21 @@ kubectl get pods -n kube-system
 ```
 
 Alternatively, you could also install the driver using helm:
+
+Add the aws-ebs-csi-driver Helm repository:
+```sh
+helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
+helm repo update
+```
+
+Then install a release of the driver using the chart
 ```sh
 helm upgrade --install aws-ebs-csi-driver \
     --namespace kube-system \
     --set enableVolumeScheduling=true \
     --set enableVolumeResizing=true \
     --set enableVolumeSnapshot=true \
-    https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/download/v0.7.1/helm-chart.tgz
+    aws-ebs-csi-driver/aws-ebs-csi-driver
```

Comment:
could you explain this syntax, having trouble understanding it. I guess the chart name is aws-ebs-csi-driver but what does it mean to have aws-ebs-csi-driver/aws-ebs-csi-driver?

 helm upgrade [RELEASE] [CHART] [flags]

Reply:
Never mind, I've got it. it is a chart reference. A/B where A is the repo name and B is the chart name.

https://helm.sh/docs/helm/helm_install/